.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: regentmarkets/debian-pg:stretch-5.26.2-96
+    steps:
+      - checkout
+      - run:
+          name: Install library dependencies
+          command: apt-get update && apt-get install -y --no-install-recommends libssl-dev zlib1g-dev pkg-config
```

Comment:
Aren't these already in the base image?

Reply:
Some packages will be failed to install if no such line: https://circleci.com/gh/binary-com/perl-WebService-Async-Onfido/15